content/page/index.en-us.md

```diff
@@ -15,6 +15,35 @@ Gitea is a painless self-hosted Git service. You can compare it with GitHub, Bit
 
 The goal of this project is to make the easiest, fastest, and most painless way of setting up a self-hosted Git service. With Go, this can be done with an independent binary distribution across ALL platforms that Go supports, including Linux, macOS and Windows, even on architectures like ARM or PowerPC.
 
+## Features
+
+- Activity timeline
+- SSH and HTTP/HTTPS protocols
+- SMTP/LDAP/Reverse proxy authentication
+- Reverse proxy with sub-path
+- Account/Organization/Repository management
+- Add/Remove repository collaborators
+- Repository/Organization webhooks (including Slack)
+- Repository Git hooks/deploy keys
+- Repository issues, pull requests and wiki
+- Migrate and mirror repository and its wiki
+- Web editor for repository files and wiki
+- Gravatar and Federated avatar with custom source
+- Mail service
+- Administration panel
+- Supports MySQL, PostgreSQL, SQLite3 and [TiDB](https://github.com/pingcap/tidb) (experimental)
+- Multi-language support ([21 languages](https://github.com/go-gitea/gitea/tree/master/options/locale))
```

Comment:
There is a new language we hasn't cherry-pick. I have counting the locale/*, it's 21.